test/inductor/test_codecache.py

```diff
@@ -35,10 +34,11 @@
     HAS_CUDA,
     HAS_GPU,
     HAS_MULTIGPU,
+    HAS_TRITON,
     requires_gpu,
+    requires_triton,
```

Comment:
Then we don't need this, right?

Reply:
This file will use this decorator. I just move it to the common file.